apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/InsertTest.cs

```diff
@@ -491,5 +491,36 @@ public void CreateLoadJob_Parquet()
             var rows = table.ListRows().ToList();
             Assert.Equal(50, rows.Count);
         }
+
+        [Fact]
+        public void CreateLoadJobWithSession()
+        {
+            string sourceUri = "gs://cloud-samples-data/bigquery/us-states/us-states.parquet";
+
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var tableRef = client.GetTableReference("_SESSION", "TempTestTable");
```

Comment:
If "_SESSION" is a magic value, perhaps we should have a constant for that?

Reply:
done